Styles/main.css

```diff
@@ -1,61 +1,92 @@
 @CHARSET "UTF-8";
 
-body{color:#801515;
-    background:#FFAAAA;
+body {
+    color: #801515;

```

Comment:
I've done a bunch of formatting in your CSS.  Note that selectors and opening `{` are on one line, and each declaration is on its own line, indented from the selector.  Finally the closing `}` is on its own line back at the same indentation level as the selector and opening `{`